resources/wikia/modules/flowTracking.js

```diff
@@ -0,0 +1,57 @@
+define('wikia.flowTracking', ['wikia.tracker', 'wikia.window', 'mw', 'jquery'], function (tracker, w, mw, $) {
+
+	var track = tracker.buildTrackingFunction({
+			category: 'flow-tracking',
+			eventName: 'flowtracking',
+			trackingMethod: 'analytics'
+
+		}),
+		userAgent = w.navigator.userAgent;
+
+	/**
+	 * Track edit flow beginning.
+	 *
+	 * @param flow name of the flow
+	 * @param extraParams additional parameters to track
+	 */
+	function beginFlow(flow, extraParams) {
+		var params = prepareParams(tracker.ACTIONS.FLOW_START, flow, extraParams);
+		track(params);
+	}
+
+	/**
+	 * Track intermediary flow step
+	 *
+	 * @param flow name of the flow
+	 * @param extraParams additional parameters to track
+	 */
+	function trackFlowStep(flow, extraParams) {
+		var params = {};
```

Comment:
Hmm, I guess Sonar is right - it's not really necessary, we could have the entire block handled in a functional way inside of isContentPage() condition